pkg/entity/installation/installation.go

```diff
@@ -0,0 +1,49 @@
+package installation
+
+import (
+	"context"
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/strmprivacy/api-definitions-go/v2/api/installations/v1"
+	"google.golang.org/grpc"
+	"strings"
+	"strmprivacy/strm/pkg/auth"
+	"strmprivacy/strm/pkg/common"
+)
+
+var client installations.InstallationsServiceClient
+var apiContext context.Context
+
+func SetupClient(clientConnection *grpc.ClientConn, ctx context.Context) {
+	apiContext = ctx
+	client = installations.NewInstallationsServiceClient(clientConnection)
+}
+
+func get(_ *cobra.Command) {
+	req := &installations.GetInstallationRequest{}
```

Comment:
I thought we added an `installation_id` parameter to GetInstallation. Are the API definitions up to date?